### PR TITLE
Add zengHou.md

### PR DESCRIPTION
**Pull request by zenghou**
- Brief resume of Zeng Hou

### CODE REVIEW

File: members/zengHou.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Lim Zeng Hou
+
+## Education 
+Faculty: USP  FASS
```
Comment (bladerail) [inline members/zengHou.md]: Might want to add Computer Science here as well

File: members/zengHou.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Major: Communications and New Media  Computer Science (pending)
```
Comment (bladerail) [inline members/zengHou.md]: Maybe include year of study?

**Review by bladerail — COMMENTED** [review]
Pretty good resume so far

**Comment by zenghou** [comment]
Necessary changes made according to review. Kindly review and merge when ready.

**Comment by bladerail** [comment]
Thank you for the changes, they have been approved. Merging now

**Pull request merged**